server/data/data-jmap/src/test/java/org/apache/james/jmap/api/filtering/RuleTest.java

```diff
@@ -100,6 +100,7 @@ void nameShouldBeMandatory() {
         assertThatThrownBy(() ->
             Rule.builder()
                 .id(UNIQUE_ID)
+                .name(null)
```

Comment:
why do you need to add this line?

Reply:
without name or with `null` are the same result but in the way i see, it make thing clear.

Reply:
that's not the way I understand the test name. But if you think this case is also interesting, please add this test case.

Reply:
Yes, i agree with more test case.
As i understand this test case name: `name` is missing, it can be either null or empty.